calico-enterprise_versioned_docs/version-3.19-1/getting-started/compatibility.mdx

```diff
@@ -145,6 +150,7 @@ Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm
 
 | {{prodname}} version | {{prodname}} support                 | Kubernetes versions |
 | -------------------- | ------------------------------------ | ------------------- |
+| 3.19                 | {{prodname}} CNI with network policy | 1.26 - 1.28         |
```

Comment:
```suggestion
| 3.19                 | {{prodname}} CNI with network policy | 1.27 - 1.29         |
```

Reply:
We didn't test 1.29 with rke2 for 3.19 ep1 because it wasn't release then. 

RKE2 has released 1.29 now but this will be updated after EP2 is out.